ClosedXML/Excel/IXLWorkbook.cs

```diff
@@ -0,0 +1,220 @@
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Globalization;
+using System.IO;
+
+namespace ClosedXML.Excel
+{
+    public interface IXLWorkbook : IDisposable
```

Comment:
I haven't added it to the developer guidelines, but would you mind installing the [CodeMaid](http://www.codemaid.net/) extension and doing a "clean-up" of this file. It will sort the methods and properties alphabetically and do some other cleaning up. I generally do this for new files that are added to the project.

Reply:
Ok I did that, but it really did not change much? Mostly just removed a bit of incorrect white space?

Reply:
Apologies, I was a bit unclear. It's the "Reorganize Active Document" (Ctrl+M, Z) that I meant. It orders the methods and properties alphabetically and according to type. The "Cleanup Active Document" (Ctrl+M, Space) is also important as I like to keep the whitespace consistent. The reason is that I often have to go way back in the logs and having clean, concise diffs make it much easier. 

I haven't had many contributions from other users, therefore the coding guidelines are poor. I have to expand on them and put them in the documentation.

![image](https://user-images.githubusercontent.com/145854/31879586-14b2387c-b7de-11e7-965a-fca62dd2d483.png)